tests/test_client_app_downloader.py

```
@@ -0,0 +1,100 @@
+
+    downloader.migrate_legacy_layout()
+
+    assert (downloads / "app" / "v2.7.13" / "asset.txt").exists()
+    assert (downloads / "app" / "v2.7.14" / "asset.txt").exists()
+    assert (downloads / "app" / "v2.7.15" / "asset.txt").exists()
+    assert (
+        downloads / "app" / "prerelease" / "v2.7.16-closed.1" / "asset.txt"
+    ).exists()
+
+
+def test_release_notes_use_single_client_app_file(downloader, tmp_path):
```

Ruff (0.15.12)

[warning] Unused function argument: `tmp_path`

(ARG001)